bash_completion

```diff
@@ -2111,33 +2111,44 @@ _included_ssh_config_files()
     local configfile i files f
     configfile=$1
 
+    # From man ssh_config:
+    # "Files without absolute paths are assumed to be in ~/.ssh if included in a user configuration
+    # file or /etc/ssh if included from the system configuration file."
+    # This behavior is not affected by the the including file location -
+    # if system configuration file is included from user's config relative includes are still
+    # resolved in user's ssh config directory.
+    local relative_include_base
+    if [[ $configfile =~ ^\/etc\/ssh.* ]]; then
+        relative_include_base="/etc/ssh"
+    else
+        relative_include_base="$HOME/.ssh"
+    fi
+
+    local depth=1
     local -a included
-    _comp_split included "$(command sed -ne 's/^[[:blank:]]*[Ii][Nn][Cc][Ll][Uu][Dd][Ee][[:blank:]]\(.*\)$/\1/p' "${configfile}")" || return
-
-    for i in "${included[@]}"; do
-        # Check the origin of $configfile to complete relative included paths on included
-        # files according to ssh_config(5):
-        #  "[...] Files without absolute paths are assumed to be in ~/.ssh if included in a user
-        #   configuration file or /etc/ssh if included from the system configuration file.[...]"
-        if ! [[ $i =~ ^\~.*|^\/.* ]]; then
-            if [[ $configfile =~ ^\/etc\/ssh.* ]]; then
-                i="/etc/ssh/$i"
-            else
-                i="$HOME/.ssh/$i"
+    local -a include_files
+    included=("$configfile")
+
+    while (("${#included[@]}" > 0)) && ((depth < 16)); do
+        depth=$((depth + 1))
```

Comment:
```suggestion
    while ((${#included[@]} > 0 && depth++ < 16)); do
```

Reply:
Thanks for updating! You kept the quoting `"..."` for `${#included[@]}` twice. Is there a reason?

Reply:
No, simply missed it. Will fix.

Reply:
Thanks